analysis/clients/nostur-yakihonne-notedeck.md

```
@@ -0,0 +1,71 @@
+# Nostur, Yakihonne, Notedeck
+
+---
+
+## Nostur (Swift, iOS)
+
+### Quick Facts
+| Setting | Value |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)